### DISCUSSION TITLE
Bug?

### DISCUSSION THREAD

**Discussion by 000XRedSide** [GitbookIO/community#617] [0. General bugs and questions] [answered]
Good morning,

I don't understand why I have a space between this block and the page
If you can help me, thanks !

![Screenshot_7](https://github.com/GitbookIO/community/assets/169863110/84aeaf88-e030-48ec-b8d0-49be88d10d78)

**Comment by thebiglabasky** [comment, marked as answer]
Hi,
This is intentional as we center the documentation on the screen right now.
